src/gui/torrentcontenttreeview.cpp

```diff
@@ -148,6 +150,28 @@ void TorrentContentTreeView::renameSelectedFile(BitTorrent::AbstractFileStorage
     }
 }
 
+void TorrentContentTreeView::mouseDoubleClickEvent(QMouseEvent *event)
+{
+    const QModelIndex index = indexAt(event->pos());
+    if (!index.isValid())
+        return;
```

Comment:
I would still call `QTreeView::mouseDoubleClickEvent(event)` in all the cases except the one when checkbox is clicked.

Reply:
Done.